backend/app/api/welfare.py

```
@@ -0,0 +1,227 @@
+        _welfare_manager = WelfareManager()
+        await _welfare_manager.connect()
+    return _welfare_manager
+
+
+# ============================================================================
+# Welfare Program Endpoints
+# ============================================================================
+
+@router.get("/programs", response_model=List[WelfareProgramResponse])
+async def get_welfare_programs(
+    category: Optional[WelfareCategory] = Query(None, description="Filter by category"),
```

Ruff (0.14.5)

Do not perform function call `Query` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)